src/css/common/shame.scss

```diff
@@ -0,0 +1,10 @@
+/*
+  Shameful overrides
+ */
+
+//semantic ui dimmer
+// WARNING: this is a temporary obviously fallacious setting that will influence dimmers on
+// ALL websites using semantic ui...
+.ui.dimmer {
```

Comment:
what's the reason it can't be scoped too?

Reply:
There is no obvious way to set our own a class for the semantic ui dimmer appearing together with modal (it might be possible to set it if you separately display a dimmer and a modal, but then again I guess it's not the way you're supposed to do it)